### DISCUSSION TITLE
The latest WSA version 2301 has been released.

### DISCUSSION THREAD

**Discussion by Ingress007** [LSPosed/MagiskOnWSALocal#384] [Q&A]
How to build wsa 2301 version?

**Comment by s1204IT** [comment]
Select release type "Insider Fast"

**Reply by Ingress007** [reply]
Why choose the type "Insider Fast"?
Microsoft App Store has been able to download version 2301.

**Reply by s1204IT** [reply]
"Release Preview" or later seems to be 2301.
If you select anything other than "Retail", you can build.

**Reply by icefrog119** [reply]
how do i keep up with the updates?

**Reply by 7gxycn08** [reply]
> how do i keep up with the updates?

monitor this page for updates and then build a new image.
[https://blogs.windows.com/windows-insider/author/windows-insider-program-team/](url)

**Reply by Almighty188** [reply]
> > how do i keep up with the updates?
> 
> monitor this page for updates and then build a new image. [https://blogs.windows.com/windows-insider/author/windows-insider-program-team/](url)

Seems like 2301.40000.7.0 is what's retail now but doesn't seem like they made any announcements of it as 2301.40000.4.0 was the last version announced.

**Comment by Almighty188** [comment]
The better way to check to see what the current version is:

https://store.rg-adguard.net/
ProductID: 9P3395VX91NR
and the choose the release type to see what the latest version for that channel is.

**Comment by Almighty188** [comment]
Current version(s) of WSA:
2302.40000.8.0 Fast Slow RP
2301.40000.7.0 Retail
